.eslintrc.js

```diff
@@ -0,0 +1,75 @@
+const sharedPlugins = [
+  'import',
+  'react',
+  'react-hooks',
+  'jest',
+  'prettier',
+  'standard',
+];
+
+const sharedExtends = [
+  'standard',
+  'plugin:react/recommended',
+  'plugin:prettier/recommended',
+  'prettier',
+  'prettier/react',
+  'prettier/standard',
+];
+
+const sharedRules = {
+  'no-multi-spaces': 'off',
+  'no-multi-str': 'off',
+  'no-mixed-operators': 'off',
+  'no-return-assign': 'off',
+  'react/react-in-jsx-scope': 'off',
+  'react-hooks/rules-of-hooks': 'error',
+  'prettier/prettier': 'error',
+};
+
```

Comment:
Is the `sharedX` stuff between flow and TS and we'll remove them once we're on one type system?

Reply:
yeah